README.md

```diff
@@ -25,10 +25,37 @@ The release page includes the full Changelog but you may also see the [changelog
 
 ## Build requirements
 * Visual Studio 2017 or newer
-* optional: CMake (when building with CMake)
+* optional: CMake (when building with CMake), Ninja
 
 ## HowTo
 You may use win_flex and win_bison directly on the command line or [use them via CustomBuildRules in VisualStudio](custom_build_rules/README.md).
+### Visual Studio Code
```

Comment:
Following the current README's and repo structure this file and images should be moved out to a separate vscode folder, linked here as done for the CustomBuildRules.  
Please also consider using "vscode" instead, because there are more binary packages then the one distributed by Microsoft.

Reply:
I'll try to mirror that directory...